apps/fz_http/lib/fz_http_web/controllers/auth_gateway_controller.ex

```diff
@@ -0,0 +1,38 @@
+defmodule FzHttpWeb.AuthGatewayController do
+  alias FzHttp.Gateways
+  use FzHttpWeb, :controller
+  require Logger
+
+  def request(conn, %{"secret" => secret}) do
```

Comment:
Would it be better if we do this as part of `GatewayChannel`? In theory, we don't need a gateway to request another HTTP(S) endpoint to register itself if we can do it when it's joining the socket.